src/applications/edu-benefits/complaint-tool/reducers/schoolSelect.js

```diff
@@ -42,7 +42,34 @@ export default function schoolSearch(state = initialState, action) {
         }
       };
     }
+    case LOAD_SCHOOLS_FAILED: {
+      const currentPageNumber = 0;
+      const institutionSelected = {};
+      const institutionQuery = action.institutionQuery;
+      const institutions = [];
+      const pagesCount = 0;
+      const searchResultsCount = 0;
+      const showInstitutions = false;
+      const showInstitutionsLoading = false;
+      const showNoResultsFound = true;
```

Comment:
Handle school loading failure by showing no results found.

Reply:
Should we distinguish no results found from unable to retrieve results due to an error? Perhaps a message such as "search is unavailable at this time, please try again later or use the manual entry fields below" and log the errors?

Reply:
I'll make a ticket for this.